tonic/models/vic/vic.py

```diff
@@ -38,7 +38,7 @@ def _get_options(self):
         self._call_vic('-o')
         return self.stdout
 
-    def run(self, global_param, logdir=None, mpi_proc=None):
+    def run(self, global_param, logdir=None):
         """run VIC"""
```

Comment:
Add `**kwargs` as an argument.  

Also, now that we're accepting other arguments, it would make since to formalize this docstring.  See the [numpy docs](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) for a good example of how to do this.  We'll at least want a short summary, parameters, and returns section.